engine/src/main/java/com/datatorrent/stram/util/LoggerUtil.java

```diff
@@ -278,4 +288,63 @@ public static synchronized void changeLoggersLevel(@Nonnull Map<String, String>
     }
     return ImmutableMap.copyOf(matchedClasses);
   }
+
+  /**
+   * Returns logger log file {@link LogFileInformation}
+   * @return logFileInformation
+   */
+  @Evolving
+  public static LogFileInformation getLogFileInformation()
+  {
+    if (shouldFetchLogFileInformation()) {
+      File logFile = new File(rootFileAppender.getFile());
+      LogFileInformation logFileInfo = new LogFileInformation(rootFileAppender.getFile(), logFile.length());
+      return logFileInfo;
+    }
+    return new LogFileInformation(null, 0);
+  }
+
+  private static FileAppender getRootFileAppender()
+  {
+    Enumeration e = LogManager.getRootLogger().getAllAppenders();
+    Set<FileAppender> fileAppenders = new HashSet<>();
+    while (e.hasMoreElements()) {
+      Appender appender = (Appender)e.nextElement();
+      if (appender instanceof FileAppender) {
+        fileAppenders.add((FileAppender)appender);
```

Comment:
For comment:
Check FileAppender log level here. There should be one (or at least one?) FileAppender with ERROR level logging threshold.

As error levels can be changed later, right now, I would restrict feature to only one FileAppender irrespective of log level. If we detect different logging level later we will enable/disable this feature accordingly.